### ISSUE TITLE
Evaluate Azure SEA with respect to Foundation

### ISSUE THREAD

**Issue by ActionAnalytics** [bcgov/cloud-pathfinder#1097]
**Describe the issue**
We want to observe the SEA and see how destructive or good fit it is for the existing foundation

**Additional context**
- After deploying the SEA
- Jon requires high-level access
- Completely blocked by Azure SEA tech work, which in turn is blocked by contract
  - HOWEVER we do have a Dev SEA and could start comparing once Jon frees up

**Definition of done**
- A demo from Warren to rest of Max, Julian and Jon, showcasing what's there and automation gaps, compare to AWS
- Create well-formed tickets once more is known

**Comment by ActionAnalytics** [comment]
This is blocked until we have an Azure SEA

**Comment by sid-gov** [comment]
Azure SEA is set up for access. Moving this to unblocked.

**Comment by wrnu** [comment]
I demoed the SEA dev environment to @ActionAnalytics and @jon-mc-git. @jon-mc-git is going to look at and compare the SEA to foundation.

**Comment by jon-mc-git** [comment]
On hold (blocked) until can get demo of what has been achieved currently in initial code deploy 'dev' environment, along with what has been learned SEA infrastructure wise when compared to AWS.  After this with Warren, will re-evaluate and create questions\review package for Warren and I to discuss further.  Next steps\action items will come from there, so we can have separate discussions on these and move forward planfully.

**Comment by wrnu** [comment]
No longer relevant
